modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,62 +1,68 @@
 @import 'atlas-variables';
```

Comment:
We no longer need to `@import 'atlas-variables';`, the `cadmin-variables` import will give us everything we need.

There is also an `analytics-reports-app.scss` file that should be isolated.